code/modules/client/client_procs.dm

```diff
@@ -1177,6 +1192,52 @@
 	var/list/screensize = getviewsize(view)
 	return max(screensize[1], screensize[2])
 
+/// Compiles a full list of verbs and sends it to the browser
+/client/proc/init_verbs()
+	if(IsAdminAdvancedProcCall())
+		return
+	var/list/verblist = list()
+	var/list/verbstoprocess = verbs.Copy()
+	if(mob)
+		verbstoprocess += mob.verbs
+		for(var/AM in mob.contents)
+			var/atom/movable/thing = AM
+			verbstoprocess += thing.verbs
+	panel_tabs.Cut() // panel_tabs get reset in init_verbs on JS side anyway
+	for(var/thing in verbstoprocess)
+		var/procpath/verb_to_init = thing
+		if(!verb_to_init)
+			continue
+		if(verb_to_init.hidden)
+			continue
+		if(!istext(verb_to_init.category))
+			continue
+		panel_tabs |= verb_to_init.category
+		verblist[++verblist.len] = list(verb_to_init.category, verb_to_init.name)
```

Comment:
```suggestion
		verblist[length(verblist)] = list(verb_to_init.category, verb_to_init.name)
```

Reply:
![image](https://media.tenor.com/IyweQyb3MhIAAAAi/the-rock-sus.gif)
Functionality change

Reply:
```suggestion
		verblist[length(verblist) + 1] = list(verb_to_init.category, verb_to_init.name)
```